viewer/packages/rendering/src/glsl/math/rand2d.glsl

```diff
@@ -1,6 +1,6 @@
+precision highp float;
```

Comment:
Is it necessary to define highp float here? Since it will imported and used by a shader which presumably has a precision identifier.

Reply:
There are files which are imported at the top of the shader file, either we need to include `precision high float` in all fragment shader or add them like this. I thought rather than adding in all fragment shaders, this would be a bit cleaner. I am open if this needs to be changed :)

Reply:
I disagree with this, and I think that all shaders should have the `precision high float` definition (not utilities). If I remember correctly, the precision needs to be at the top / first statement in the shader. So what happens if f.ex. the import is moved or simply removed for some reason? That should not affect the precision. What happens if a shader declares medium precision, but imports this utility?